content/millennium/r4/documents/document-reference.md

```diff
@@ -69,13 +69,22 @@ _Implementation Notes_
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12724066
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12742400
 
 #### Response
 
 <%= headers status: 200 %>
 <%= json(:R4_DOCUMENT_REFERENCE_BUNDLE) %>
 
+#### Patient Authorization Request
+
+    GET https://fhir-myrecord.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12742400
```

Comment:
This should be pointing to ehr
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference?patient=12742400

Reply:
addressed